cmd/server/handlers/server/metrics.go

```diff
@@ -36,9 +37,15 @@ func PrepareMetricsValues(s Storager, m storage.Monitor) {
 		metricValue, _ := strconv.ParseFloat(fmt.Sprintf("%v", v.Field(i).Interface()), 64)
 
 		if typeOfS.Field(i).Name == "PollCount" {
-			s.AddCounter(metricID, models.Counter(1))
+			err := s.AddCounter(metricID, models.Counter(1))
+			if err != nil {
+				logger.WriteErrorLog(err.Error(), "Counter")
+			}
 		} else {
-			s.SetGauge(metricID, models.Gauge(metricValue))
+			err := s.SetGauge(metricID, models.Gauge(metricValue))
+			if err != nil {
+				logger.WriteErrorLog(err.Error(), "Gauge")
```

Comment:
```suggestion
				logger.WriteErrorLog(err.Error(), "Gauge")
				return err
```
Если есть возможность поймать и отправить ошибку наверх, необходимо поймать и отправить. Тем более, что данная функция, в частности, дергается внутри хендлера и при наличии ошибки должна честно выдавать клиенту 500.